web-locks/release-following-termination.tentative.https.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
```

Comment:
nit: `<html>` `<head>` and `<body>` are optional in HTML5. The document will parse fine without them.

Reply:
It will parse fine, but without a body, the script will have nowhere to place the iframe. I'll keep that tag and remove the others.